lib/shopifyHmac/constructHmacParameter.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const crypto = require('crypto');
+
+const constructHmacParameter = (req, options) =>
```

Comment:
I would give this (and the file) a more specific name, because it is not just a generic HMAC parameter we are reconstructing but a shopify one
```suggestion
const constructShopifySignature = (req, options) =>
```